templates/zerver/help/create-a-stream.md

```diff
@@ -18,45 +18,52 @@ sidebar.  Streams can be either:
-and can be no longer than 60 characters.
+5. Enter the title of your stream in the **Stream name** input.
+If you designate your stream as public, the **Stream name** will be
+displayed under the Streams on the left sidebar. Your stream name must
+be unique to all other stream names and can be no longer than
+60 characters.
 
     !!! tip ""
-        You can optionally enter a brief description of your stream in **Stream description**
-        to give other users a general idea of what's being
-        discussed in your stream. The description can be seen under the stream
-        name in the stream overview.
+        You can optionally enter a brief description of your stream in
+        **Stream description** to give other users a general idea of
+        what's being discussed in your stream. The description can be
+        seen under the stream name in the stream overview.
 
-7. The **Stream accessibility** option controls the privacy of your stream. There are two options:
-    - **Anybody can join** - This option makes your stream **public** and
-    accessible to all users.
+7. The **Stream accessibility** option controls the privacy of your
+stream.There are two options:
+    * **Anybody can join** - This option makes your stream **public**
+    and accessible to all users.
 
-    - **People must be invited** - This option makes your stream **private**.
-    Only users you invite will be able to access this stream.
+    * **People must be invited** - This option makes your stream
+    **private**. Only users you invite will be able to access this stream.
 
-8. If your stream is public, you can choose to alert users about the new stream
-by clicking the **Announce stream** checkbox. Users who have been added to the
-new stream will always be notified, but if the **Announce stream** feature is
-enabled, all users will be notified of the stream's creation.
+8. If your stream is public, you can choose to alert users about the new
+stream by clicking the **Announce stream** checkbox. Users who have been
+added to the new stream will always be notified, but if the
+**Announce stream** feature is enabled, all users will be notified
+of the stream's creation.
 
-9. To automatically subscribe a user to your stream, scroll down to **People to add**
-and tick the checkboxes with the names of the users you want to add.
+9. To automatically subscribe a user to your stream, scroll down to
+**People to add** and tick the checkboxes with the names of the users
+you want to add.
 
     !!! tip ""
-        To search for specific users, enter their usernames in the **Filter names** box.
+        To search for specific users, enter their usernames in the
+        **Filter names** box.
 
 10. Once you are ready to create your stream, click the **Create** button
-located at the bottom right of the window. The stream will now appear in the
-left sidebar for the users that you subscribed.
+located at the bottom right of the window. The stream will now appear
```

Comment:
> the bottom right

Keep in mind that the UI may change — don’t describe it in more detail than is needed.